backend/src/main/java/com/woowacourse/moragora/domain/meeting/Meeting.java

```diff
@@ -83,6 +73,28 @@ public long calculateTardy() {
                 .sum();
     }
 
+    public long tardyCountByUserId(final Long loginUserId) {
```

Comment:
```suggestion
    public long tardyCountByUserId(final Long userId) {
```
요 method에서는 어떤 사용자의 id인지까지는 몰라도 되지 않을까 싶네요

Reply:
맞아요, Meeting 입장에서는 "로그인"이란 키워드를 몰라도 될 것 같네요 :)